modules/nw-multus-macvlan-object.adoc

```diff
@@ -40,6 +40,10 @@ The following object describes the configuration parameters for the macvlan CNI
 |`string`
 |Optional: The maximum transmission unit (MTU) to the specified value. The default value is automatically set by the kernel.
 
+|`linkInContainer`
+|`boolean`
+|Optional: Specifies if the master interface is in the container network namespace or the main network namespace.
```

Comment:
Same question/suggestion as my comment on line 51 of modules/nw-multus-ipvlan-object.adoc

Reply:
Revised to make description clearer hopefully